otlp/src/otlp/logs/component.cpp

```diff
@@ -38,6 +45,8 @@ LoggerComponent::LoggerComponent(const components::ComponentConfig& config, cons
 
     LoggerConfig logger_config;
     logger_config.max_queue_size = config["max-queue-size"].As<size_t>(65535);
+    logger_config.max_attempts = ClampAttempts(config["max-attempts"].As<size_t>(1));
```

Comment:
validate >0 here, not in CLampAttempts()

Reply:
\+